src/tvmlkit.cs

```diff
@@ -819,8 +826,9 @@ interface TVViewElement : NSCopying {
 			set;
 		}
 
+		[Internal][Sealed]
 		[Export ("updateType")]
-		TVElementUpdateType UpdateType { get; }
+		TVElementUpdateType _UpdateType { get; }
```

Comment:
only for `!XAMCORE_4_0` ?

Reply:
Actually the workaround should continue to exist in XAMCORE_4_0 (theoretically until tvOS 12 is the earliest tvOS version we support). I've updated the manually bound property accordingly.